### PR TITLE
Add change_toolchain_version script

### PR DESCRIPTION
**Pull request by bgurney-rh**
Supersedes: #210 
Related: #191

### CODE REVIEW

**Comment by bgurney-rh** [comment]
It seems to work for me, after arriving back from the weekend, and running it on the devicemapper-rs repo:

```
$ ./change_toolchain_version.py lowest ../../devicemapper-rs/.github/workflows/main.yml ../../devicemapper-rs/.github/workflows/main.yml 1.49.0 1.53.0
```

After that, I can checkout a new branch for a PR, run `git add` on the file, and create a commit.

File: misc_scripts/change_toolchain_version.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        search_key = KEY_LSRT
+    elif args.toolchain == "current":
+        search_key = KEY_CDRT
+
+    old_verstring = args.old_version + r"  " + search_key
+    new_verstring = args.new_version + r"  " + search_key
+
+    output = search_file(search_key, old_verstring, new_verstring, filename)
+
+    # for line in output:
```
Comment (mulkieran) [inline misc_scripts/change_toolchain_version.py:69]: Plz drop debug stuff

**Review by mulkieran — CHANGES_REQUESTED** [review]
Just one thing...

File: misc_scripts/change_toolchain_version.py (hunk 1 above)
Comment (bgurney-rh) [reply misc_scripts/change_toolchain_version.py]: Removed and rebased.

**Review by mulkieran — APPROVED** [review]

**Pull request merged**